### PR TITLE
Windows installation instructions (Chocolatey)

### PR DESCRIPTION
**Pull request by avdaredevil**
## About
Updated readme to include instructions for installing on windows with Choco

#### based on awesome work by @digitalcoyote in #268 (making this since that PR is now Stale)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ this new Go implementation by downloading the binaries from the [**Releases page
 - [with MacPorts (macOS)](#macports-macos)
 - [with apt (Debian)](#apt-debian)
 - [with pacman (Arch Linux)](#pacman-arch-linux)
+- [with Choco (Windows)](#windows-installation-using-choco)
```
Comment (ahmetb) [inline README.md:80]: Chocolatey is the right term I believe.

**Review by ahmetb — CHANGES_REQUESTED** [review]
Thank you.

**Comment by avdaredevil** [comment]
Done!

File: README.md [hunk 2]
```diff
@@ -140,6 +141,12 @@ Available as official Arch Linux package. Install it via:
 sudo pacman -S kubectx
 ```
 
+### Windows Installation (using Choco)
```
Comment (ahmetb) [inline README.md:144]: Chocolatey

File: README.md [hunk 3]
```diff
@@ -77,6 +77,7 @@ this new Go implementation by downloading the binaries from the [**Releases page
 - [with MacPorts (macOS)](#macports-macos)
 - [with apt (Debian)](#apt-debian)
 - [with pacman (Arch Linux)](#pacman-arch-linux)
+- [with Chocolatey (Windows)](#windows-installation-using-choco)
```
Comment (ahmetb) [inline README.md:80]: -chocolatey

**Review by ahmetb — CHANGES_REQUESTED** [review]

**Pull request merged**